docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+
+We use the [GameCI](https://game.ci/) project as a foundation for running GitHub CI/CD builds for Unity. The GameCI project provides a collection of GitHub Actions that are used to build Unity projects.
+
+#### 1. Create a new branch in your project called `qa-tests`.
+
+#### 2. Follow the steps on GameCI to activate a [Unity License](https://game.ci/docs/github/activation).
+
+It is critical that you follow this step! This allows the CI/CD Docker images to properly start and use Unity.
+You can test that the license is configured properly by running this [GitHub Action](https://github.com/marketplace/actions/unity-activate)
+in your project.
+
+#### 3. Add the QA Bot workflow file to your project.
+
+The QA Bot workflow file is a GitHub Action that will build your Unity project and start a test that will run a bot. From step 2 above,
+you should have a `.github/workflows` folder. Inside of this folder, create a new file called `qa-bot.yml` and paste the contents below.
+
+```yaml
+#file: .github/workflows/qa-bot.yml
+
+name: Regression Games Bots QA 🤖
+
+on: [push]
+
+# See https://game.ci/docs/github/test-runner and RG_DOCS for more info
+jobs:
+  build:
+    name: Build my project ✨
+    runs-on: ubuntu-latest
+    permissions: write-all # This is to allow writing results into the artifacts section of GitHub Actions
+    steps:
+          
+      # Checkout (without LFS)
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Git LFS
+      - name: Create LFS file list
+        run: git lfs ls-files -l | cut -d' ' -f1 | sort > .lfs-assets-id
+
+      - name: Restore LFS cache
+        uses: actions/cache@v3
+        id: lfs-cache
+        with:
+          path: .git/lfs
+          key: ${{ runner.os }}-lfs-${{ hashFiles('.lfs-assets-id') }}
+
+      - name: Git LFS Pull
+        run: |
+          git lfs pull
+          git add .
+          git reset --hard
+
+      # Cache
+      - uses: actions/cache@v3
+        with:
+          path: Library
+          key: Library-${{ hashFiles('Assets/**', 'Packages/**', 'ProjectSettings/**') }}
+          restore-keys: |
+            Library-
+
+      # Test
+      # See https://game.ci/docs/github/test-runner for more information
+      - name: Run tests
+        uses: game-ci/unity-test-runner@v2
+        env:
+          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+        with:
+          customParameters: -RG_BOT ${{ secrets.RG_BOT }} -RG_API_KEY ${{ secrets.RG_API_KEY }} -RG_HOST ${{ secrets.RG_HOST }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+      # Output
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Build
+          path: build
+```
+
+:::info
+
+Note that this will actually run all of your tests, not just the QA Bot test. To configure this workflow to only
+run the QA test, see the [Unity Test Runner Command Line docs](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/reference-command-line.html),
```

Comment:
In the interest of making setup as quick as possible, it'd be nice to have an example of how you'd do this for a hypothetical project. Ideally something where I'd substitute <my bot test directory> or <my other test directories>